tests/test_prior.py

```diff
@@ -67,12 +105,12 @@ def log_likelihood(self):
         return (self.theta + 5)**2
 
 
-def test_map_contains_log_det_jacobian():
+def test_map_invariance_to_transform():
     m1 = DummyModel(with_transform=True)
     m2 = DummyModel(with_transform=False)
     assert np.allclose(m1.log_marginal_likelihood().numpy(),
-                       m2.log_marginal_likelihood().numpy() + m1.log_scale), \
-                       "MAP objective should differ by log|Jacobian| of the transform"
+                       m2.log_marginal_likelihood().numpy()), \
+                       "MAP objective should not be affected by a transform"
```

Comment:
Could you add some explanation (not in this part of the code, but perhaps in the PR description - so that it ends up in the git log when merging) why we completely change this around?

Reply:
Yep, I've updated the description to explain this change and that of the mcmc helper.